notebooks/gammanu.ipynb

```diff
@@ -0,0 +1,1105 @@
+{
```

Comment:
<p>"Some implementations will fail at t=0, so let's re-evaluate the integral for that special case:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore-research/pyscf-ipu/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>